### PR TITLE
Support P4Studio 9.5.2

### PR DESCRIPTION
**Pull request by Yi-Tseng**
Add P4Studio 9.5.2 support

### CODE REVIEW

File: stratum/hal/lib/barefoot/bf_sde_wrapper.cc [hunk 1]
```diff
@@ -1648,7 +1648,7 @@ std::string BfSdeWrapper::GetBfChipType(int device) const {
 std::string BfSdeWrapper::GetSdeVersion() const {
 #if defined(SDE_9_3_1)
   return "9.3.1";
-#elif defined(SDE_9_5_0)
+#elif defined(SDE_9_5_0) || defined(SDE_9_5_2)
   return "9.5.0";
```
Comment (pudelkoM) [inline stratum/hal/lib/barefoot/bf_sde_wrapper.cc:1652]: I don't think that's smart. We should return the correct string.

**Comment by pudelkoM** [comment]
Update the `stratum/tools/release-stratum.sh` script as well?

File: stratum/hal/lib/barefoot/bf_sde_wrapper.cc (hunk 1 above)
Comment (Yi-Tseng) [reply stratum/hal/lib/barefoot/bf_sde_wrapper.cc:1652]: oops.
I use "search and replace" tool to change this file.
Let me fix that...

**Comment by bocon13** [comment]
Should we just replace 9.5.0 with 9.5.2? Or would you prefer to remove 9.5.0 in a future commit?

**Comment by Yi-Tseng** [comment]
> Should we just replace 9.5.0 with 9.5.2? Or would you prefer to remove 9.5.0 in a future commit?

Maybe we can remove this in the next release? I can add a comment in the README

**Review by pudelkoM — APPROVED** [review]

**Pull request merged**